samples/charts/data-chart/polar-series/Pages/DataChartPolarSeries.razor

```diff
@@ -0,0 +1,244 @@
+        <select value=@SeriesType
+                @onchange="OnSeriesTypeChanged">
+            <option>Area</option>
+            <option>Spline Area</option>
+            <option>Spline</option>
+            <option>Line</option>
+            <option>Scatter</option>
+        </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
+        <DataChart Height="100%" Width="100%"
+                   ChartTitle="Sailing Chart"
+                   Subtitle="Wind Speed vs. Boat Speed"
+                   TitleTopMargin="10"
+                   IsHorizontalZoomEnabled="true"
+                   IsVerticalZoomEnabled="true"
+                   Legend="@LegendRef"
+                   @ref="DataChartRef">
+
+            <NumericAngleAxis Name="angleAxis"
+                                 StartAngleOffset=-90
+                                 Interval=30
+                                 MinimumValue=0
+                                 MaximumValue=360 />
+            <NumericRadiusAxis Name="radiusAxis"
+                                  InnerRadiusExtentScale=0.1
+                                  RadiusExtentScale=0.9
+                                  MinimumValue=0
+                                  MaximumValue=100
+                                  Interval=25 />
+
+        </DataChart>
+        }
+    </div>
+</div>
+
+@code {
+
+    private List<SamplePolarItem> DataSource;
+    private DataChart dataChart;
+    private DataChart DataChartRef
+    {
+        get { return dataChart; }
+        set
+        {
+            dataChart = value;
+            this.setSeries("Spline");
+            StateHasChanged();
+        }
+    }
+
+    private Legend legend;
+    private Legend LegendRef
+    {
+        get { return legend; }
+        set
+        {
+            legend = value;
+            StateHasChanged();
+        }
+    }
+    private string seriesType;
+    private string SeriesType
+    {
+        get { return seriesType; }
+        set
+        {
+            seriesType = value;
+            this.setSeries("Spline");
+            StateHasChanged();
+        }
+    }
+
+    protected override void OnInitialized()
+    {
+        DataChartCoreModule.Register(IgniteUIBlazor);
+        DataChartPolarCoreModule.Register(IgniteUIBlazor);
+        DataChartPolarModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+
+        LegendModule.Register(IgniteUIBlazor);
+        DataSource = SamplePolarData.Create();
+    }
+
+    public void OnSeriesTypeChanged(ChangeEventArgs e)
+    {
+        var selectedSeries = e.Value.ToString();
+        SeriesType = selectedSeries;
+        this.setSeries(selectedSeries);
+    }
+
+    public void setSeries(string seriesType)
+    {
```

Comment:
there should be no code-behind to create series/axes and all these series/axes must be created in blazor code